lodash/index.d.ts

```diff
@@ -11202,7 +11202,7 @@ declare namespace _ {
     }
 
     //_.cloneDeepWith
-    type CloneDeepWithCustomizer<TValue, TResult> = (value: TValue) => TResult;
+    type CloneDeepWithCustomizer<TValue, TResult> = (value: TValue, key?: number|string, object?: any, stack?: any) => TResult;
```

Comment:
This is a proper use then. Usually that is mistaken but in this case it is correct.